appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -1739,7 +1662,7 @@ Blockly.ReplMgr.putAsset = function(projectid, filename, blob, success, fail, fo
         return false;
     if (!force && (top.ReplState.state != this.rsState.ASSET && top.ReplState.state != this.rsState.CONNECTED))
         return false;           // We didn't really do anything
-    if (top.usewebrtc) {
+    if (!force) {               // Force is only used for updating the emulator
         // Note: if using webrtc we ignore the passed in callback and
         // instead just call makeAssetTransferred ourselves
```

Comment:
Technically this comment is no longer true since all assets use this path now.

Reply:
Fixed